### PR TITLE
:sparkles::lock: add github action scanning

### PR DESCRIPTION
**Pull request by m4nch0t**
Add support for github actions sbom

### CODE REVIEW

File: tests/.github/workflows/install-python.yaml
Comment (tcarmet) [inline tests/.github/workflows/install-python.yaml:1]: This seems like an unrelated file that was committed. Or if it's not unrelated, the runner do not exist and will not be picked up.

**Review by tcarmet — CHANGES_REQUESTED** [review]
One comment requiring some attention, the rest LGTM

File: tests/.github/workflows/install-python.yaml
Comment (m4nch0t) [reply tests/.github/workflows/install-python.yaml:1]: It's only a test file for sbom generation. To check if scan are running well.

Comment (tcarmet) [reply tests/.github/workflows/install-python.yaml:1]: ok ok, my bad then.

**Review by tcarmet — APPROVED** [review]

**Pull request merged**